internal/core/command/types_test.go

```diff
@@ -58,7 +57,6 @@ func (MockBody) Close() error {
 func TestExecute(t *testing.T) {
 	expectedResponseBody := "Sample Response Body"
 	expectedResponseCode := http.StatusOK
-	LoggingClient = logger.MockLogger{}
```

Comment:
Commented here; applies elsewhere.

You're removing this.  Isn't a logger.LoggingClient needed for this test?